is_core/generic_views/table_views.py

```diff
@@ -81,6 +82,12 @@ def _get_header_order_by(self, model, full_field_name):
         return order_by
 
     def _get_header_label(self, model, field_name):
+        method = getattr(self, '_get_{}_label'.format(field_name), None)
+        if method and callable(method):
+            return method()
+        elif self.field_headers and field_name in self.field_headers.keys():
+            return self.field_headers[field_name]
+
```

Comment:
Ještě to teda musím předělat pro ten rekurzivní přístup k fieldům, např. 'order__eshop'